events/tests.py

```diff
@@ -49,6 +49,129 @@ def test_event_plaintext(self):
         self.assertEqual(response['Content-Type'], 'text/plain')
 
 
+class RemoteAddrTest(TransactionTestCase):
```

Comment:
that's a lot of testing for very little functionality, which gives me a "bad feeling" generally and esp. since it was done by a bot.

could we not have a "tighter fit" around what's actually happening?